rhobs/alerting/data_plane/prometheus.exporter_alerts.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: konflux-exporter-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: exporter_alerts
+    interval: 5m
```

Comment:
I don't think we should use non-default interval here. It will needlessly make it more complex.